### ISSUE TITLE
A logged in user should not be asked to login/register again?

### ISSUE THREAD

**Issue by gitp18** [bradtraversy/node_passport_login#30]
After successful sign up,
http://localhost:5000/users/login and http://localhost:5000/users/register should redirect to http://localhost:5000/dashboard page, until the user sign out.

**Comment by bradtraversy** [comment]
You are right. If you want to make a pull request I will check it out and accept

**Comment by harmnot** [comment]
I am using sequelize and it doesn't save the session, after I login and I go to login router again, it still on login form,

**Comment by CloudBranch** [comment]
Another thing to improve the UX would be to log the user in automatically after registering

in "routes/users.js" add this code

`req.login(user, function(err) {
                  if (err) { return next(err); }
                  return res.redirect('/dashboard');
                });`

**Comment by CloudBranch** [comment]
`        bcrypt.genSalt(10, (err, salt) => {
          bcrypt.hash(newUser.password, salt, (err, hash) => {
            if (err) throw err;
            newUser.password = hash;
            newUser
              .save()
              .then(user => {

                req.login(user, function(err) {
                  if (err) { return next(err); }
                  return res.redirect('/dashboard');
                });

              })
              .catch(err => console.log(err));
          });
        });`

**Comment by hackrx** [comment]
If somebody finds solution for this then please comment out here

**Comment by bradtraversy** [comment]
Closing as part of stale issue cleanup. The repo has since been updated with current EJS syntax and dependency/security fixes, so this issue is likely obsolete. If this still happens against the latest master, please open a fresh issue with updated reproduction steps.
